docs/api/js/accounts.md

```diff
@@ -288,14 +285,14 @@ async withdraw(transaction: {
 }): Promise<TransactionResponse>
 ```
 
-| Name                     | Description                                                         |
-| ------------------------ | ------------------------------------------------------------------- |
-| tx.to                    | The address of the recipient on L1.                                 |
-| tx.amount                | The amount of the token to transfer.                                |
-| token (optional)         | The address of the token. `ETH` by default.                         |
-| bridgeAddress (optional) | The address of the bridge contract to be used.                      |
-| overrides (optional)     | Transaction overrides, such as `nonce`, `gasLimit` etc. |
-| returns                  | A `TransactionResponse` object                                      |
+| Name                     | Description                                             |
+| ------------------------ | ------------------------------------------------------- |
+| tx.to                    | The address of the recipient on L1.                     |
+| tx.amount                | The amount of the token to transfer.                    |
+| token?         | The address of the token. `ETH` by default.             |
+| bridgeAddress? | The address of the bridge contract to be used.          |
+| overrides?     | **Ethereum** transaction overrides. May be used to pass `gasLimit`, `gasPrice`, etc.    |
```

Comment:
Because withdraw is l2 transaction it has no l1 interaction for the user. 

The only withdrawal-related interaction l1 is done with the withdrawal finalizer.